### DISCUSSION TITLE
CubeCoders Application Management Panel

### DISCUSSION THREAD

**Discussion by ChrLau** [selfhst/icons#96] [New Icon Request]
Hi, CudeCoders AMP is a self-hostable Gameserver Manager.
Website: https://cubecoders.com/AMP

I wasn't able to find a good logo file on their site (it's not the favicon), but their subreddit has one:
https://new.reddit.com/r/Amp/
https://styles.redditmedia.com/t5_2rjge/styles/communityIcon_jgl9zq94d4u31.png

**Comment by shollyethan** [comment]
I'm working on finding a better quality image and will respond here when I've found something.

**Reply by ChrLau** [reply]
If you wish I can ask on their official Discord.

**Reply by mkopnsrc** [reply]
This seems littler higher on quality, https://minecraft.erfworld.com/Images/LogoColor.png

**Reply by shollyethan** [reply]
> This seems littler higher on quality, https://minecraft.erfworld.com/Images/LogoColor.png

This was perfect - thanks! CubeCoders AMP has now been added to the collection.
